src/components/CommonComponents/Pagination/Page.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { MouseEvent, ReactElement, FC } from 'react';
+import classnames from 'classnames';
+import { LocalizedLink as Link } from 'gatsby-theme-i18n';
+import { FormattedMessage } from 'react-intl';
+import { PageModel, PageType } from './util/buildPaginationModel';
+
+interface Props {
+  className: string;
+  page: PageModel;
+  hrefBuilder: (n: number) => string;
+  onPageChange?: (e: MouseEvent) => void;
+}
+
+const getPagerContent = (page: PageModel): string | ReactElement => {
```

Comment:
Also why is the name of the function `getPager`? What would a Pager mean?

Reply:
fixed